dask/base.py

```diff
@@ -386,6 +399,25 @@ def visualize(*args, **kwargs):
     for d in dsks:
         dsk.update(d)
 
+    color = kwargs.get('color')
+
+    if color == 'order':
+        from .order import order
+        o = order(dsk)
+        try:
+            cmap = kwargs.pop('cmap')
+        except KeyError:
+            import matplotlib.pyplot as plt
+            cmap = plt.cm.viridis
```

Comment:
Since we use bokeh elsewhere, perhaps pull the cmap from bokeh instead of matplotlib to remain consistent? e.g. `https://github.com/dask/dask/blob/master/dask/diagnostics/profile_visualize.py#L135`

Reply:
Could also use `import_required` here to provide a nice message, idk. I personally have no issue with import errors, but others have in the past.

Reply:
I'm inclined to stay with matplotlib as the default plotting solution when all you need is plotting.  I find that it is currently more widely deployed and more commonly understood.  I certainly prefer bokeh for interactive applications, but it doesn't seem necessary here.

I'm also inclined to wait on the import_required bit until someone complains.

Reply:
I'm still inclined to go with matplotlib as the default, just because of the current install base.